src/Pim/Bundle/EnrichBundle/Controller/ProductController.php

```diff
@@ -127,7 +127,7 @@ public function __construct(
      *
      * @return array
      */
-    public function indexAction()
+    public function indexAction(Request $request)
```

Comment:
Yes, this method is overrriden on EE. As signature is not the same, there is a warning on EE